src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -85,12 +85,13 @@ public class HttpBridge extends AbstractVerticle implements HealthCheckable {
     /**
      * Constructor
      *
-     * @param bridgeConfig bridge configuration
+     * @param bridgeConfig    bridge configuration
      * @param metricsReporter MetricsReporter instance for scraping metrics from different registries
      */
     public HttpBridge(BridgeConfig bridgeConfig, MetricsReporter metricsReporter) {
         this.bridgeConfig = bridgeConfig;
         this.metricsReporter = metricsReporter;
+        System.out.println(this.bridgeConfig.getHttpConfig().isConsumerEnabled());
```

Comment:
Thsi probably either should not be here or it should be a proper log message?

Reply:
Agree.

Reply:
removed unwanted system out